src/app.rs

```diff
@@ -314,5 +317,8 @@ fn usage() -> HashMap<&'static str, Help> {
         , "Limit results based on modification time within the duration provided:\n    \
              using a duration: <NUM>d <NUM>h <NUM>m <NUM>s (e.g. 10h, 1d, 35min...)\n    \
              or a date and time: YYYY-MM-DD HH:MM:SS");
+    doc!(h, "verbose"
+        , "Print errors to stdout"
```

Comment:
(this should be `stderr` instead of `stdout`) I would suggest the following alternative help texts:

short:
- "Enable display of filesystem errors"

long:
- "Enable the display of filesystem errors for situations such as insufficient permissions or dead symlinks."